Refresh.GameServer/Database/GameDatabaseContext.Playlists.cs

```diff
@@ -132,17 +179,35 @@ public IEnumerable<GamePlaylist> GetPlaylistsByAuthorContainingPlaylist(GameUser
             .Where(p => p.Publisher.UserId == user.UserId)
             .Where(p => !p.IsRoot);
 
-    public IEnumerable<GameLevel> GetLevelsInPlaylist(GamePlaylist playlist, TokenGame game) =>
+    public IEnumerable<GameLevel> GetLevelsInPlaylist(GamePlaylist playlist, TokenGame game)
+    {
         // TODO: When we have postgres, remove the `AsEnumerable` call for performance. 
+        IEnumerable<LevelPlaylistRelation> relations = this.LevelPlaylistRelations
+            .Where(l => l.Playlist == playlist).AsEnumerable();
+
+        // Only sort by order if needed, to improve performance
+        if (game == TokenGame.LittleBigPlanet3)
+            relations = relations.OrderBy(r => r.Index);
```

Comment:
So you are suggesting to remove custom level ordering as a whole and to just not support that feature, because of potential playlist support in other games using level overrides? I am not understanding the link between the two.